src/coffee/mapping.coffee

```diff
@@ -112,6 +113,51 @@ class Mapping
 
     categories
 
+  # parses the categoryOrderHints column for a given row
+  mapCategoryOrderHints: (rawMaster, rowIndex) ->
+    catOrderHints = {}
+    # check if there actually is something to parse in the column
+    return catOrderHints unless @hasValidValueForHeader(rawMaster, CONS.HEADER_CATEGORY_ORDER_HINTS)
+    # parse the value to get a list of all catOrderHints
+    rawCatOrderHints = rawMaster[@header.toIndex CONS.HEADER_CATEGORY_ORDER_HINTS].split GLOBALS.DELIM_MULTI_VALUE
+    for rawCatOrderHint in rawCatOrderHints
```

Comment:
I honestly just copy pasted this from the method above... Should I go for consistency or quality of new code?